hackathon_site/registration/jinja2/registration/emails/waitlisted_email_body.html

```diff
@@ -0,0 +1,11 @@
+<p>Hello {{ user.first_name|striptags }},</p>
+
+<p>The {{hackathon_name}} team has reviewed your application, and due to both the high caliber and high volume of applications, we have decided to defer your application to the next round. You will hear back from us by {{final_review_response_date}}regarding a final application decision.</p>
```

Comment:
Space missing after `final_review_response_date` (another case of needing a space inside the tag for consistency too)